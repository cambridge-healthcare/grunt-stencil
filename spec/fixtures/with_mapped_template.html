This page has a mapped template.