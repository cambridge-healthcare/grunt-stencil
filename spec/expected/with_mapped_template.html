This is a template. This page has a mapped template.
